src/scss/partials/_nav.scss

```diff
@@ -343,10 +345,10 @@ $nav-popover-padding          : spacing(x2);
             transition: all $nav-transition-duration ease-in-out;
         }
         &:before {
-            top: -($nav-toggleIcon-space + $nav-toggleIcon-height);
```

Comment:
Is the gap between the lines 6px is the new design?  Or is it 5px plus the 1px height – if the latter then it'd be better to change the spacing variable down 1px and keep these lines as adding the height + space.

Reply:
Ah yeah - that makes more sense!